ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/RpcApis.java

```diff
@@ -42,8 +44,8 @@
       return Optional.of(WEB3);
     } else if (name.equals(ADMIN.getCliValue())) {
       return Optional.of(ADMIN);
-    } else if (name.equals(PERM.getCliValue())) {
-      return Optional.of(PERM);
```

Comment:
PERM relates to the permissioning APIs so these should stay and EEA should be added.

A conversation at EEA needs to take place if the permissioning APIs can be included in the EEA namespace as currently they are reserved for the specified privacy ones.

Reply:
False alarm - a duplicate PERM has been replaced.  ;)